web/shapeworks/public/forms/optimize.json

```diff
@@ -10,41 +10,50 @@
                 "number_of_particles": {
                     "type": "integer",
                     "default": 128,
-                    "title": "Number of particles"
+                    "title": "Number of particles",
+                    "description": "Number of correspondence points to generate"
                 },
                 "relative_weighting": {
                     "type": "number",
                     "default": 0.05,
                     "title": "Relative weighting",
-                    "x-cols": 6
+                    "x-cols": 6,
+                    "description": "Relative weighting of correspondence term during optimization"
                 },
                 "initial_relative_weighting": {
                     "type": "number",
                     "default": 0.05,
                     "title": "Initial",
-                    "x-cols": 6
+                    "x-cols": 6,
+                    "x-class": "float-right pl-5",
+                    "description": "Relative weighting of correspondence term during initialization"
                 },
                 "starting_regularization": {
                     "type": "integer",
                     "default": 1000,
                     "title": "Starting regularization",
-                    "x-cols": 6
+                    "x-cols": 6,
+                    "description": "Starting regularization of correspondence covariance matrix"
                 },
                 "ending_regularization": {
                     "type": "integer",
                     "default": 10,
                     "title": "Ending regularization",
-                    "x-cols": 6
+                    "x-cols": 6,
+                    "x-class": "float-right pl-5",
+                    "description": "Ending regularization of correspondence covariance matrix"
                 },
                 "iterations_per_split": {
                     "type": "integer",
                     "default": 1000,
-                    "title": "Iterations per split"
+                    "title": "Iterations per split",
+                    "description": "Number of iterations for each particle split"
                 },
                 "optimization_iterations": {
                     "type": "integer",
                     "default": 1000,
-                    "title": "Optimization iterations"
+                    "title": "Optimization iterations",
+                    "description": "Number of optimizations to run"
                 },
                 "use_landmarks": {
                     "type": "boolean",
```

Comment:
Do we have a description for landmarks?

Reply:
No, that entry was left blank: https://docs.google.com/document/d/1gbyEg2t4yKgBDfc9kPeBP3WRfcl7pvJcL95JaaiggTY/edit?usp=sharing